logstash/metadata.csv

```diff
@@ -52,3 +52,8 @@ logstash.pipeline.plugins.outputs.events.duration_in_millis,gauge,,millisecond,,
 logstash.pipeline.plugins.filters.events.in,gauge,,,,Number of events into the filter plugin.,0,logstash,filters events in
 logstash.pipeline.plugins.filters.events.out,gauge,,,,Number of events out from the filter plugin.,0,logstash,filters events out
 logstash.pipeline.plugins.filters.events.duration_in_millis,gauge,,millisecond,,Duration of events in the filter plugin.,0,logstash,filters events duration
+logstash.pipeline.queue.capacity.max_queue_size_in_bytes,gauge,,byte,,Max queue capacity in bytes,0,logstash,queue capacity max queue size in bytes
+logstash.pipeline.queue.capacity.max_unread_events,gauge,,,,Maximum unread events allowed in queue,0,logstash,queue capacity max unread events
+logstash.pipeline.queue.capacity.page_capacity_in_bytes,gauge,,byte,,Queue page capacity in bytes,0,logstash,queue capacity max page size
+logstash.pipeline.queue.capacity.queue_size_in_bytes,gauge,,byte,,Queue disk used in bytes,0,logstash,queue capacity queue size in bytes
+logstash.pipeline.queue.events,gauge,,,,Number of events in the queue,0,logstash,queue events
```

Comment:
We should make the usage documentation clear that these metrics are only valid and available for `persistent` queue type pipelines and will not be present for queue types of type `memory` - just to avoid confusion and set correct expectations when using this integration.